identify/extensions.py

```diff
@@ -203,3 +203,7 @@
     'Rakefile': EXTENSIONS['rb'],
     'setup.cfg': EXTENSIONS['ini'],
 }
+
+NAMES_IGNORE_EXTENSION = {
+    'CMakeLists.txt': EXTENSIONS['cmake'],
```

Comment:
I believe the smaller change is to add `CMakeLists.txt` to `NAMES` -- is there a reason that doesn't work?

Reply:
Thank you for the fast response!

It would work, but it would result in CMakeLists.txt files being tagged with 'plain-text' while a *.cmake file would not have such a tag.  
I believe CMakeLists.txt and *.cmake files should be treated the same, but I understand if you don't want to add the mapping for just this one special case. 

However, I would be fine with just adding `CMakeLists.txt` to `NAMES`. I will submit a new commit if you want.

Reply:
yes, let's do that